src/main/java/com/epam/preprod/sirenko/Demo.java

```diff
@@ -1,25 +1,36 @@
 package com.epam.preprod.sirenko;
 
 import com.epam.preprod.sirenko.containers.CommandContainer;
-import com.epam.preprod.sirenko.services.impl.SerializationServiceImpl;
+import com.epam.preprod.sirenko.containers.FactoryContainer;
+import com.epam.preprod.sirenko.strategy.Strategy;
+import com.epam.preprod.sirenko.strategy.impl.AutoGeneratedProductCreationStrategy;
+import com.epam.preprod.sirenko.strategy.impl.ManualProductCreationStrategy;
 import com.epam.preprod.sirenko.util.ConsoleReader;
+import com.epam.preprod.sirenko.util.PrintToConsole;
+import com.epam.preprod.sirenko.util.ValidatorOfConsoleInput;
 
 import java.io.*;
 
 public class Demo {
-	private SerializationServiceImpl serializationServiceImpl;
-	
-	public Demo(SerializationServiceImpl serializationServiceImpl) {
-		this.serializationServiceImpl = serializationServiceImpl;
-		init();
-	}
-	
-	private void init() {
-		serializationServiceImpl.serializeProductsListFromFile();
-	}
+	private static final String CHOOSE_STRATEGY = "To create product print 1, to autogenerate product print 0";
+	private static final String INCORRECT_INPUT_ONE_OR_ZERO = "Please print only 0 or 1";
 	
 	public static void main(String[] args) throws IOException {
-		CommandContainer commandContainer = new CommandContainer();
+		PrintToConsole.printString(CHOOSE_STRATEGY);
+		String inputStrategy = ConsoleReader.readFromConsole();
+		if (!ValidatorOfConsoleInput.checkInputStringIsNumberOneOrZero(inputStrategy)) {
+			PrintToConsole.printString(INCORRECT_INPUT_ONE_OR_ZERO);
+			inputStrategy = ConsoleReader.readFromConsole();
+		}
+		Strategy strategy;
```

Comment:
Can be moved to private method for better readability

Reply:
done